app/controllers/user_notifications_controller.rb

```diff
@@ -21,4 +21,28 @@ def delete
     ).destroy!
     render nothing: true
   end
+
```

Comment:
Can you come up with better names for these functions?
- `unsubscribe` can also apply to the other function.
- `stop_this_notification` isn't very clear either. What `notification` is it talking about?

If someone new to the codebase reads these function names, we want it to be easy for the reader to distinguish between these two names.